core/commands/quick_stage.py

```diff
@@ -83,18 +87,29 @@ def get_menu_options(self):
         """
         status_entries = self.get_status()
         menu_options = []
+        staged_count = 0
+        unstaged_count = 0
 
         for entry in status_entries:
+            if entry.index_status!="?" and entry.index_status!=" ":
+                staged_count += 1
             if entry.working_status in ("M", "D", "?"):
+                unstaged_count += 1
                 filename = (entry.path if not entry.index_status == "R"
                             else entry.path + " <- " + entry.path_alt)
                 menu_text = "[{0}] {1}".format(entry.working_status, filename)
                 menu_options.append(MenuOption(True, menu_text, filename, entry.index_status == "?"))
 
-        if not menu_options:
-            return [MenuOption(False, CLEAN_WORKING_DIR, None, None)]
+        if not status_entries:
+            menu_options.append(MenuOption(False, CLEAN_WORKING_DIR, None, None))
+            return menu_options
```

Comment:
The change to the `if` line is okay, but change these two lines back to:
```python
return [MenuOption(False, CLEAN_WORKING_DIR, None, None)]
```
And actually, please move this block to the top of the method to return early, immediately after the definition of `status_entries`.